share/completions/readelf.fish

```diff
@@ -0,0 +1,75 @@
+	.got\t"Global offset table"
+	.hash\t"Symbol hash table"
+	.interp\t"Path name of a program interpreter"
+	.plt\t"Procedure linking table"
+	.shstrtab\t"String table for the section header table names"
+	.strtab\t"String table"
+	.symtab\t"Symbol table"
+'
+
+complete -c readelf -s a -l all -d "Equivalent to -h -l -S -s -r -d -V -A -I"
+complete -c readelf -s h -l file-header -d "Display the ELF file header"
+complete -c readelf -s l -l program-headers -l segments -d "Display the program headers"
+complete -c readelf -s S -l section-headers -l sections -d "Display the sections' header"
+complete -c readelf -s g -l section-groups -d "Display the section groups"
+complete -c readelf -s t -l section-details -d "Display the section details"
+complete -c readelf -s e -l headers -d "Equivalent to -h -l -S"
+complete -c readelf -s s -l syms -l symbols -d "Display the symbol table"
+complete -c readelf -l dyn-syms -d "Display the dynamic symbol table"
+complete -c readelf -l lto-syms -d "Display LTO symbol tables"
+complete -c readelf -l sym-base -d "Force base for symbol sizes" -a '0\t"Mixed" 8\t"Octal" 10\t"Decimal" 16\t"Hexadecimal"' -x
+complete -c readelf -s C -l demangle -d "Decode mangled/processed symbol names" -a 'none auto gnu-v3 java gnat dlang rust' -x
+complete -c readelf -l no-demangle -d "Don't demangle low-level symbol names"
+complete -c readelf -l recurse-limit -d "Enable a demangling recursion limit"
+complete -c readelf -l no-recurse-limit -d "Disable a demangling recursion limit"
+complete -c readelf -s U -l unicode -d "Controls the display of non-ASCII characters in identifier names" -a 'd\t"default" l\t"locale" e\t"escape" x\t"hex" h\t"highlight" i\t"invalid"' -x
+complete -c readelf -s n -l notes -d "Display the core notes"
+complete -c readelf -s r -l relocs -d "Display the relocations"
+complete -c readelf -s u -l unwind -d "Display the unwind info"
+complete -c readelf -s d -l dynamic -d "Display the dynamic section"
+complete -c readelf -s V -l version-info -d "Display the version sections"
+complete -c readelf -s A -l arch-specific -d "Display architecture specific information"
+complete -c readelf -s c -l archive-index -d "Display the symbol/file index in an archive"
+complete -c readelf -s D -l use-dynamic -d "Use the dynamic section info when displaying symbols"
+complete -c readelf -s L -l lint -l enable-checks -d "Display warning messages for possible problems"
+complete -c readelf -s x -l hex-dump -d "Dump the contents of specified section as bytes" -a $sections -x
+complete -c readelf -s p -l string-dump -d "Dump the contents of specified section as strings" -a $sections -x
+complete -c readelf -s R -l relocated-dump -d "Dump the relocated contents of of specified section" -a $sections -x
+complete -c readelf -s z -l decompress -d "Decompress section before dumping it"
+for c in {a,A,r,c,L,f,F,g,i,o,m,p,t,R,l,s,O,u,T,U}
+	complete -c readelf -o w$c -d "Display the contents of DWARF debug sections"
```

Comment:
we can simplify the loop:

```fish
for c in a A r c L f F g i o m p t R l s O u T U
```

Actually it could be helpful to add to the description the specific section, for example

	complete -o wa -e 'Display the contents of the .debug_abbrev section'

Reply:
> Actually it could be helpful to add to the description the specific section, for example
Yeah, I agree with it.